Source/Extensions/StringExtensions.swift

```diff
@@ -2,59 +2,28 @@ import Foundation
 import UIKit
 
 extension String {
-
-    func getBundle(forClass: AnyClass) -> Foundation.Bundle {
-#if SWIFT_PACKAGE
-        let baseBundle = Bundle.module
-#else
-        let baseBundle = Foundation.Bundle(for: forClass)
-#endif
-        let path = baseBundle.path(forResource: "Frames", ofType: "bundle")
-        // swiftlint:disable:next force_unwrapping
-        return path == nil ? baseBundle : Foundation.Bundle(path: path!)!
+    func localized(comment: String = "") -> String {
+        NSLocalizedString(self, bundle: Bundle.base, comment: "")
```

Comment:
If we take the method argument (comment), we should also pass it

```suggestion
        NSLocalizedString(self, bundle: Bundle.base, comment: comment)
```